### ISSUE TITLE
error message for required: true failure is odd

### ISSUE THREAD

**Issue by kevinburkeshyp** [balderdashy/sails-postgresql#190]
here's an example error message:

```
Invalid attributes sent to TableName:
 • userId
   • `undefined` should be a text (instead of "null", which is a object)
   • "required" validation rule failed for input: null
```

- Not sure null should be considered an object.
- Believe `undefined` should be the name of a column or similar, but it isn't getting populated properly.

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburkeshyp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by particlebanana** [comment]
Discussed in more detail here: https://github.com/sailsjs/anchor/issues/100. Thanks @kevinburkeshyp!
